app/src/main/java/org/oppia/app/topic/review/ReviewSelector.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.topic.review
+
+import org.oppia.app.model.Subtopic
+
+/** Listener for when a skill is selected for review. */
```

Comment:
`/** Listener for when a subtopic is selected for review. */`

Reply:
+1

Reply:
Updated.